src/TLuaInterpreter.h

```diff
@@ -538,6 +538,7 @@ public slots:
 
     std::list<std::string> mCaptureGroupList;
     std::list<int> mCaptureGroupPosList;
+    std::list<int> mCaptureGroupUtf16PosList;
     std::list<std::list<std::string>> mMultiCaptureGroupList;
```

Comment:
Is it possible that you also need a `mMultiCaptureGroupUtf16List` as well - or is it that only the Lua sub-system has to worry about the `multiMatches` table and it is not relevant to the C++ core - though IIRC the text does get shown in the *Central Debug Console*? I guess we will check to see whether the debug output from using "AND/multiline" triggers works...

Reply:
If I understand it correctly, `mMultiCaptureGroupUtf16List` shouldn't be needed, because I calculate correct UTF-16 length by converting values from `mMultiCaptureGroupUtf16List` to QString and then getting correct length from there.